examples/mixture-mpi.py

```diff
@@ -97,10 +100,10 @@ def main(ctx_factory=cl.create_some_context):
     local_mesh, global_nelements = generate_and_distribute_mesh(comm, generate_mesh)
     local_nelements = local_mesh.nelements
 
-    discr = EagerDGDiscretization(
+    dcoll = DiscretizationCollection(
         actx, local_mesh, order=order, mpi_communicator=comm
     )
-    nodes = thaw(actx, discr.nodes())
+    nodes = thaw(op.nodes(dcoll), actx)
```

Comment:
Can't we have a `dcoll.nodes()` interface?

Reply:
`dcoll.discr_from_dd("vol").nodes()`?

Reply:
> `dcoll.discr_from_dd("vol").nodes()`?

Yeah that's basically what `op.nodes(dcoll, dd=None)` does

Reply:
We should also make `nodes` always return a `thaw`ed object

Reply:
But then you have to pass in an array context.

Reply:
Maybe let's just not freeze instead? https://github.com/inducer/grudge/blob/thg/remove-symbolics/grudge/op.py#L202

Reply:
Actually, this would be problematic for lazy, right?

Reply:
@MTCam I've gone ahead and updated everything to use `dcoll.nodes(...)` and `dcoll.normal(...)`